### ISSUE TITLE
Move from SQL to http API

### ISSUE THREAD

**Issue by Suor** [idrdex/star_api#2]
As far as I understand you aim to make this usable by extended group of people or even anyone. Providing database access to all of them could be both insecure and clunky, so we can use common solution for this scenario - create http API at stargeo.io, so that people could access needed data without database access. 

Later, some tasks as starting analysis or annotating things could also be implemented via http API for easier automation, see for example [this](https://github.com/idrdex/star-django/issues/22).

**Comment by iDrDex** [comment]
Yes.  We need the ability to issue analyses via the API.  As well as query results from API.  I have basic data queries implemented in this repo for now.

**Comment by zhilongjia** [comment]
I think a REST API for stargeo will be a great way to analyse and annotate data, which is easy to use and independent from a certain programming language, such as Python. Thank you.

**Comment by iDrDex** [comment]
I think this is the plan in due course.
